src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.test.tsx

```diff
@@ -81,3 +81,25 @@ it('Render a loading state checking search disabled', () => {
 
   expect(queryByPlaceholderText('Filter by name/url')).toHaveAttribute('disabled');
 });
+
+it('finds search box, enters text, and checks text occurrence', async () => {
+  (useRepositoryParams as jest.Mock).mockImplementation(() => ({ isLoading: false }));
+  (usePopularRepositoriesQuery as jest.Mock).mockImplementation(() => ({
+    isLoading: false,
+    data: { data: [defaultPopularRepository], meta: { count: 1, limit: 20, offset: 0 } },
+  }));
+
+  const { getByPlaceholderText, queryByText } = render(
+    <ReactQueryTestWrapper>
+      <PopularRepositoriesTable />
+    </ReactQueryTestWrapper>,
+  );
+
+  const searchBox = getByPlaceholderText('Filter by name/url');
+  fireEvent.change(searchBox, { target: { value: 'yourSearchText' } });
+
+  waitFor(() => {
+    const occurrences = queryByText('yourSearchText', { exact: false });
+    expect(occurrences).toBeTruthy();
```

Comment:
This is starting to look good!  
But you need to find all "occurrences". In this case I believe you can use the "query**All**ByText". 

That method would return an array of items that are a match.

Currently if there is more than one item found you'd likely get an error that the test doesn't know which one you are wanting to target.

So once you have an array of items, you can check the length of that array (array.length) and have the expect look for a length of 2. 
There are a couple ways to do that with "expect" so I'll leave that for you to figure out.